dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Would be good to also move this to `.dev/docker/`. We can document all this in the developer manual `doc/`.